File: program133.cpp
/*
OOP program

OUTPUT:

Enter the number:
4
Factorial is:24


*/
#include<iostream>
using namespace std;

int Factorial(int iNo)
{
	int iFact=1;
	int iCnt=0;
	
	for(iCnt=iNo;iCnt>=1;iCnt--)
	{
		iFact=iFact*iCnt;
	}
	
	return iFact;
}

int main()
{
	int iValue=0;
	int iRet=0;
	
	cout<<"Enter the number:\n";
	cin>>iValue;
	
	iRet=Factorial(iValue);
	cout<<"Factorial is:"<<iRet<<endl;
	return 0;
}

File: program133.java
/*
OUTPUT:

Enter the number:
222435
Entered number is:222435
4       2       2       2


Enter the number:
-12456
Entered number is:-12456
6       4       2

*/

import java.lang.*;
import java.util.*;

class Number
{
	private int iNo;
	
	public void Accept()
	{
		Scanner sobj=new Scanner(System.in);
		System.out.println("Enter the number:");
		this.iNo=sobj.nextInt();
		
	}
	
	public void Display()
	{
		System.out.println("Entered number is:"+this.iNo);
	}
	
	public void EvenDigit()
	{
		int iDigit=0;
		
		if(iNo<0)
		{
			iNo=-iNo;
		}
		
		while(iNo!=0)
		{
			iDigit=iNo%10;
			if(iDigit%2==0)
			{
				System.out.print(iDigit+"\t");
			}
			iNo=iNo/10;
		}
	}
}

class program133
{
	public static void main(String arg[])
	{
		Number nobj=new Number();
		nobj.Accept();
		nobj.Display();
		nobj.EvenDigit();
		
	}
}
